packages/order/src/migrations/Migration20240219102530.ts

```diff
@@ -0,0 +1,462 @@
+import { generatePostgresAlterColummnIfExistStatement } from "@medusajs/utils"
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20240219102530 extends Migration {
```

Comment:
Have there been any changes to the model, or is it pretty much the same with v1?

Reply:
There are changes to the current model. the `order` entity is using less fields and adding a `deleted_at` on it. The other tables are either the same or new so they won't conflict.